changelog/upgrade-notes.md

```diff
@@ -31,6 +31,10 @@ but with additional css:
 }
 ```
 
+#### Removal of `ol.source.TileVector`
+
+The experimental `ol.source.TileVector` class has been removed. Its functionality is now provided by `ol.source.VectorTile`. The API is the same, except for the `tileLoadFunction`, which is no longer an `ol.TileVectorLoadFunctionType`, but an `ol.TileLoadFunctionType`.
+
```

Comment:
Searching could be solved probably by putting vectortiles somewhere in the example tags? Or mentioning it in the API docs explicitly.